theano/tensor/tests/test_basic.py

```diff
@@ -1892,10 +1886,7 @@ def test_py_c_match():
     normal=(rand_ranged(1, 10, (2, 3)),
             numpy.asarray(1, dtype=config.floatX)),
     empty=(numpy.asarray([], dtype=config.floatX),
-           numpy.asarray(1, dtype=config.floatX)),
-    # The test seems to fail!!
-    int=(randint_ranged(1, 10, (2, 3)),
-            numpy.asarray(1, dtype=config.floatX)))
+           numpy.asarray(1, dtype=config.floatX)))
```

Comment:
What error did you got with this? scipy.stats.chi2.sf() work with uint8 and int8 input.